.github/workflows/docker-test-containers.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         include:
           - source: jaegertracing/all-in-one:1.17
             target_tag: jaeger
-          - source: otel/opentelemetry-collector-dev:latest
+          - source: otel/opentelemetry-collector-contrib-dev:latest
```

Comment:
is "latest" a good thing to be using here? We've seen quite a bit of collector drift over the past year, which could end up causing unexpected test failures. Just wondering if we should start targeting a fixed version, rather than always grabbing latest.